Snippets/Gateway/Gateway_2/UpgradeGuide/2to3/UpgradeGuideFeatureUsage.cs

```diff
@@ -0,0 +1,14 @@
+﻿using NServiceBus;
+using NServiceBus.Features;
+
+class UpgradeGuideUsage
+{
+    void EnableGatewayBefore(EndpointConfiguration endpointConfiguration)
+    {
+        #region 2to3EnableGatewayBefore
+
+        endpointConfiguration.EnableFeature<Gateway>();
```

Comment:
can the docs engine mix snippets from the v2 project and the v3 project into a single document?

Reply:
I'm not sure but moving it resolved in not having that strange workaround to not build the projects.

Reply:
Tested and seems to work fine.